packages/website/src/components/KitShowcases.astro

```diff
@@ -0,0 +1,32 @@
+---
+import ShowcaseLinks from './ShowcaseLinks.astro';
+const { showcases, hasShowcase } = Astro.props;
+// max-w-[198px] max-h-[177px]
```

Comment:
Is this commented code going to be used later on?

Reply:
....not honestly sure. lol This is a completely new component so my guess is it was put here during testing? Things are mostly good how they are now so went ahead and removed it. 👍🏼